### PR TITLE
Update the docstring for `HydrostaticFreeSurfaceModel`

### PR DESCRIPTION
**Pull request by simone-silvestri**

### CODE REVIEW

**Review by navidcy — APPROVED** [review]

File: src/Models/HydrostaticFreeSurfaceModels/hydrostatic_free_surface_model.jl [hunk 1]
```diff
@@ -93,6 +94,8 @@ Keyword arguments
                preallocated `CenterField`s.
   - `forcing`: `NamedTuple` of user-defined forcing functions that contribute to solution tendencies.
   - `closure`: The turbulence closure for `model`. See `Oceananigans.TurbulenceClosures`.
+  - `timestepper`: A symbol that specifies the time-stepping method. 
+                   Either `:QuasiAdamsBashforth2` (default) or `:SplitRungeKutta3`.
```
Comment (glwagner) [inline src/Models/HydrostaticFreeSurfaceModels/hydrostatic_free_surface_model.jl:98]: Why isn't the AB2 scheme "split"? Weirded out that we use the same name for AB2 for nonhydrostatic/hydrostatic but a different name for RK3.

Comment (navidcy) [reply src/Models/HydrostaticFreeSurfaceModels/hydrostatic_free_surface_model.jl:98]: Just noticed this. I suggest to remove the split from the timestepper name?

Comment (simone-silvestri) [reply src/Models/HydrostaticFreeSurfaceModels/hydrostatic_free_surface_model.jl:98]: The scheme is called like that, as opposed to the `:RungeKutta3` scheme for the non-hydrostatic model. Are you suggesting a name change to the method or only in the constructor?

Comment (glwagner) [reply src/Models/HydrostaticFreeSurfaceModels/hydrostatic_free_surface_model.jl:98]: Why is RK3 "split" but AB2 is not?

**Pull request merged**